cmd/edit/cluster/cmd.go

```diff
@@ -639,6 +639,11 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 	if clusterRegistryConfigArgs != nil {
+		prompt := "Enabling cluster registry will cause rollout across all nodepools. Do you want to proceed?"
```

Comment:
change text to:

`Warning: Enabling or editing the cluster registry configuration will cause a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node). Do you want to proceed?`